docs/hyper-params.ipynb

```diff
@@ -0,0 +1,1600 @@
+{
```

Comment:
<p>What is <code>mul</code>  ?</p><p>Maybe give an example that matches the text above, i.e. <code>accuracy</code>  ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/791/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>